README.md

```diff
@@ -1,16 +1,16 @@
 # Top.gg Node SDK
 
-An official module for interacting with the Top.<span>gg API
+An official module for interacting with the Top.gg API
 
 # Installation
 
 `yarn add @top-gg/sdk` or `npm i @top-gg/sdk`
 
 # Introduction
 
-The base client is Topgg.Api, and it takes your Top.<span>gg token and provides you with plenty of methods to interact with the API.
+The base client is Topgg.Api, and it takes your Top.gg token and provides you with plenty of methods to interact with the API.
 
-Your Top.<span>gg token can be found at `top.gg/bot/(BOT_ID)/webhooks` and copying the token.
```

Comment:
These spans are here to avoid creating a link to Top.gg

Reply:
What's creating a link to it?

Reply:
> What's creating a link to it?

One of the docs sites we use doesn't parse markdown correctly and turns them into links like [Top.gg](https://top.gg)

Reply:
I see, does it also create a link if you escape with a backslash? I made this change because a non-terminated `<span>` tag causes TypeDoc to generate semantically incorrect HTML.

Reply:
IIRC it will turn into a literal "\\." , I tried it and obviously didn't go with it
Let's leave it how it is and we can move away from rtfd like I want to anyways, at which point we can fix it.

Reply:
![image](https://user-images.githubusercontent.com/7953076/226073607-973e5856-cf22-497d-a3ea-9e3fdd8413c9.png)